pkg/registry/clusteroperator/machineset/storage.go

```diff
@@ -136,11 +136,11 @@ func NewStorage(opts etcd.Options) (nodegroups, nodegroupsStatus rest.Storage) {
 		// Used to match objects based on labels/fields for list.
 		PredicateFunc: Match,
 		// DefaultQualifiedResource should always be plural
-		DefaultQualifiedResource: clusteroperator.Resource("clusterservicenodegroups"),
+		DefaultQualifiedResource: clusteroperator.Resource("clusterservicemachinesets"),
```

Comment:
Likewise here. This should be "machinesets" instead of "clusterservicemachinesets".